tests/jquery.testHelper.js

```diff
@@ -227,7 +227,7 @@ $.testHelper = {
 			}, 10000 );
 
 			// bind the recursive call to the event
-			( self.eventTarget || $.mobile.pageContainer ).one( event, function( event, data ) {
+			( self.eventTarget || $( ".ui-pagecontainer" ) ).one( event, function( event, data ) {
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?